src/move/emojicoin_dot_fun/sources/hex_codes.move

```diff
@@ -0,0 +1,2380 @@
+    }
+
+    // Because the bytecode changes when the length of the SYMBOL vector changes, we will simply
+    // hardcode the number of bytes to be 11. The first byte will be the signifier of how long
+    // the symbol is, called `SYMBOL_LEN` the next `SYMBOL_LEN` bytes will be the symbol,
+    // and the rest of the bytes will be 0s and ignored.
+    public(friend) fun encode_symbol(
```

Comment:
- [ ] inline
- [ ] Define the constant in the file where is needed
- [ ] Define test-only func `get_EXPECTED_FULL_SYMBOL_LENGTH`
- [ ] Write a test in the calling file that compare's its constant with the one in this file